compose/ui/ui-text/src/jsMain/kotlin/androidx/compose/ui/text/Cache.js.kt

```diff
@@ -16,11 +16,9 @@
 
 package androidx.compose.ui.text
 
-// expire cache entries after `expireAfter` after last access
-internal class ExpireAfterAccessCache<K, V>() : Cache<K, V> {
-    internal val map = HashMap<K, V>()
+// TODO Use WeakMap once available https://youtrack.jetbrains.com/issue/KT-44309
+internal actual typealias WeakKeysCache<K, V> = NoCache<K, V>
```

Comment:
Here is an example of how to use JS WeakMap from kotlin/js:

Declare an external interface:
https://github.com/JetBrains/compose-multiplatform/blob/08d5fa3487791f766b02a9e4084340516128c55f/html/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/builders/InputAttrsScope.kt

Usage:
https://github.com/JetBrains/compose-multiplatform/blob/08d5fa3487791f766b02a9e4084340516128c55f/html/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/builders/InternalControlledInputUtils.kt

Reply:
I saw the `external` way, but it's better to do it in separate PR